src/app/store/task/task.effects.ts

```diff
@@ -0,0 +1,59 @@
+import { Injectable } from "@angular/core"
+
+import { of, tap } from "rxjs"
+import { catchError, map, mergeMap, concatMap } from "rxjs"
+
+import { createEffect, Actions, ofType } from "@ngrx/effects"
+import * as taskActions from "./task.actions"
+import { Task } from "../../models/task.model"
+import { Router } from "@angular/router"
+import { HttpClient } from '@angular/common/http'
+
+import { tasksApiUrl } from '../../utils/constants'
+
+@Injectable()
+export class TaskEffects {
+    constructor(private actions$: Actions, private router: Router, private http: HttpClient) { }
+
+    loadTasks$ = createEffect(() => this.actions$.pipe(
+        ofType(taskActions.loadTasks),
+        mergeMap(() => this.http.get<Task[]>(tasksApiUrl).pipe(
```

Comment:
instead of rewriting the http calls here, inject the necessary service and use the methods from the service

Reply:
also seems like you're not making multiple calls to BE so instead of mergeMap you can use switchMap